lego/apps/users/models.py

```diff
@@ -522,3 +531,48 @@ def ignore_date(date):
         elif winter_to < (date.month, date.day) < winter_from:
             return False
         return True
+
+
+class PhotoConsent(BasisModel):
+    user = models.ForeignKey(
+        User, related_name="photo_consents", on_delete=models.CASCADE
+    )
+    semester = models.CharField(max_length=6)
```

Comment:
This should be constrained with some choices. As we have upgraded to django 3 now, you can also use the new enumeration types provided by Django[1]

[1]: https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types